src/bin/vip-dev-env-purge.js

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
```

Comment:
Please make sure that this file has the execute bit set; otherwise, it will not work in Linux (GitHub does not show file attributes or maybe I cannot find them).